extensions/stackdriver/log/logger.cc

```diff
@@ -226,6 +217,41 @@ bool Logger::exportLogEntry(bool is_on_done) {
   return true;
 }
 
+void Logger::addTCPLabelsToLogEntry(
+    const ::Wasm::Common::RequestInfo& request_info,
+    google::logging::v2::LogEntry* log_entry) {
+  auto label_map = log_entry->mutable_labels();
+  (*label_map)["tcp_remote_ip"] = request_info.source_address;
```

Comment:
Do we need `tcp` prefix? What did we use in mixer v1?

Reply:
Changed them to match mixer v1